crates/subspace-networking/src/behavior/persistent_parameters.rs

```diff
@@ -51,6 +51,9 @@ pub trait NetworkingParametersRegistry: Send + Sync {
     /// It removes p2p-protocol suffix.
     async fn next_known_addresses_batch(&mut self) -> Vec<PeerAddress>;
 
+    /// Reset the inner batcher to the initial state.
+    fn reset_batcher(&mut self);
```

Comment:
This seems to be the first "batcher" mention in the interface and it is implementation-specific thing. Let's find a more generic name here to prevent confusion and provide a default implementation with an empty function making it optional to implement.